lib/IRGen/IRGen.cpp

```diff
@@ -176,22 +176,24 @@ void swift::performLLVMOptimizations(IRGenOptions &Opts, llvm::Module *Module,
                            addSwiftContractPass);
   }
 
-  if (Opts.Sanitize == SanitizerKind::Address) {
+  if (Opts.Sanitizers & SanitizerKind::Address) {
     PMBuilder.addExtension(PassManagerBuilder::EP_OptimizerLast,
                            addAddressSanitizerPasses);
     PMBuilder.addExtension(PassManagerBuilder::EP_EnabledOnOptLevel0,
                            addAddressSanitizerPasses);
   }
   
-  if (Opts.Sanitize == SanitizerKind::Thread) {
+  if (Opts.Sanitizers & SanitizerKind::Thread) {
     PMBuilder.addExtension(PassManagerBuilder::EP_OptimizerLast,
                            addThreadSanitizerPass);
     PMBuilder.addExtension(PassManagerBuilder::EP_EnabledOnOptLevel0,
                            addThreadSanitizerPass);
   }
 
   if (Opts.SanitizeCoverage.CoverageType !=
-      llvm::SanitizerCoverageOptions::SCK_None) {
+      llvm::SanitizerCoverageOptions::SCK_None ||
+      (Opts.Sanitizers & SanitizerKind::Fuzzer)) {
```

Comment:
@jrose-apple It's a bit convoluted.
You mean in `CompilerInvocation.cpp`?  The code there sets `FrontendOptions`, but not `IRGenOptions`.